hack/cr.yaml

```diff
@@ -2,17 +2,15 @@ apiVersion: "logging.openshift.io/v1alpha1"
 kind: "ClusterLogging"
 metadata:
   name: "example"
```

Comment:
can we change this to 'logging' or 'cluster-logging' since there is no reason the 'example' cant be what we recommend

Reply:
given that this is under `hack`, I personally would like to leave it as "example" to discourage customers using this file since customers should be using OLM to create this normally.

Reply:
I think we should make sure that the CR created by OLM doesn't have `example` as its name... i would agree that `cluster-logging` is the most appropriate name there

Reply:
We'll address on next PR to OLM to update with the lastest CR bits